fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/event/Event.java

```diff
@@ -13,17 +15,19 @@ public long getStartTimestampInMillis() {
 
   public abstract long getEndTimestamp();
 
+  public abstract MutableLiveData<EventState> getState();
+
   public long getEndTimestampInMillis() {
     return getEndTimestamp() * 1000;
   }
 
   @Override
   public int compareTo(Event o) {
-    int start = Long.compare(this.getStartTimestamp(), o.getStartTimestamp());
+    int start = Long.compare(o.getStartTimestamp(), this.getStartTimestamp());
```

Comment:
Purpose is to have the latest events on top which is how sorting list of events is usually implemented